app/controllers/scientists_controller.rb

```diff
@@ -0,0 +1,22 @@
+class ScientistsController < ApplicationController
+  def show
+    @scientist = Scientist.find(params[:id])
+  end
+
+  def destroy
+    # Find the scientist in question based on `experiment_id`
+    @scientist = Scientist.find(params[:id])
+    # If incoming params includes `experiment_id`
+    if params[:experiment_id]
+      # Sets `experiment` variable to instance of Experiment class
+        # found by searching its ID
+      experiment = Experiment.find(params[:experiment_id])
+      # Calls specific instance of `experiment` from this scientist
+        # and destroys it (destroys instance of scientist_experiment with this association)
+      @scientist.experiments.delete(experiment)
+      redirect_to "/scientists/#{@scientist.id}"
+    else
+      redirect_to "/scientists/#{@scientist.id}"
+    end
+  end
```

Comment:
We spoke about this over Slack a little but but thinking about what operation on the database we are performing, we are deleting a row from the joins table, therefore this controller would not be the correct controller to handle this operation. You can think of controllers as facades that facilitate the behavior or their counterpart database tables.

If I want to facilitate work on the scientists table, then the ScientistsController is the best place for that work to reside. Since we are not actually doing any work on the scientists table, then this controller wold not be the best choice.